lib/experimental/fonts/font-library/font-library.php

```diff
@@ -82,3 +82,52 @@ function wp_unregister_font_collection( $collection_id ) {
 );
 
 wp_register_font_collection( $default_font_collection );
+
+// @core-merge: This code should probably go into Core's src/wp-includes/functions.php.
+if ( ! function_exists( 'wp_get_font_dir' ) ) {
+	/**
+	 * Returns an array containing the current fonts upload directory's path and URL.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param array $defaults {
+	 *     Array of information about the upload directory.
+	 *
+	 *     @type string       $path    Base directory and subdirectory or full path to the fonts upload directory.
+	 *     @type string       $url     Base URL and subdirectory or absolute URL to the fonts upload directory.
+	 *     @type string       $subdir  Subdirectory
+	 *     @type string       $basedir Path without subdir.
+	 *     @type string       $baseurl URL path without subdir.
+	 *     @type string|false $error   False or error message.
+	 * }
+	 *
+	 * @return array $defaults {
+	 *     Array of information about the upload directory.
+	 *
+	 *     @type string       $path    Base directory and subdirectory or full path to the fonts upload directory.
+	 *     @type string       $url     Base URL and subdirectory or absolute URL to the fonts upload directory.
+	 *     @type string       $subdir  Subdirectory
```

Comment:
Yes, I think we should include the `subdir` property to match `wp_get_upload_dir()` because we are using the output of this function to update `upload_dir` filter before uploading fonts. Currently, we are not using it in the font library related code.  It is used just in core's `_wp_upload_dir` to set the `/year/month` sub directory:

https://github.com/WordPress/wordpress-develop/blob/ddb67a0099c9912c46a22374bcf90e5aac2f25e3/src/wp-includes/functions.php#L2503-L2524